agent/api/task/task.go

```diff
@@ -719,6 +720,19 @@ func (task *Task) initializeContainersV3MetadataEndpoint(uuidProvider utils.UUID
 	}
 }
 
+// initializeContainersV4MetadataEndpoint generates an v4 endpoint id which we reuse the v3 container id (they are the same) for each container, constructs the
```

Comment:
could you break this comment into multiple lines?

Reply:
yeah, will do